presto-main/src/main/java/com/facebook/presto/operator/SimplePagesHashStrategy.java

```diff
@@ -33,6 +34,9 @@
         implements PagesHashStrategy
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(SimplePagesHashStrategy.class).instanceSize();
+    private static final int OBJECT_ARRAY_LIST_INSTANCE_SIZE = ClassLayout.parseClass(ObjectArrayList.class).instanceSize();
+    private static final int IMMUTABLE_LIST_HEADER_SIZE = ClassLayout.parseClass(ImmutableList.class).headerSize();
```

Comment:
`ImmutableList` is abstract, so is this better than `parseClass(Object.class).headerSize()`? (except for documentation purposes)

(no change requested)